README.md

```diff
@@ -98,7 +101,7 @@ If the command is typed into a channel, either the `[server name]`, the `[channe
 /ssound sound H:\rain.wav match (pitter patter)
 ```
 
-This is convenient, because the plugin only sees the [hostname][] (like a URL) of the specific server that's hosting the IRC channel, and not what might show up in the HexChat Network List.
+This is convenient, because the plugin only sees the [hostname][] (Server URL) of the specific server that's hosting the IRC channel, and not what might show up in the HexChat Network List.

```

Comment:
How would I indicate that the hostname HexChat uses is the one reported by the server?

For example, `test.example.com` can be reported by the server, but won't be how the server is identified *anywhere* in HexChat's UI, as the server was connected to by IP.

Reply:
Added an information section at the end of the README with an overview.